oak_functions_sdk/src/lib.rs

```diff
@@ -14,17 +14,12 @@
 // limitations under the License.
 //
 
-//! SDK functionality that provides idiomatic Rust wrappers around the underlying Oak Functions
-//! platform functionality.
+#![doc = include_str!("../README.md")]
 
 use oak_functions_abi::{proto::OakStatus, StorageGetItemResponse};
 use std::convert::AsRef;
 
-/// Reads and returns the user request.
-///
-/// This function is idempotent. Multiple calls to this function all return the same value.
-///
-/// See [`read_request`](https://github.com/project-oak/oak/blob/main/docs/oak_functions_abi.md#read_request).
+/// See [`read_request`](https://github.com/project-oak/oak/blob/main/docs/oak_functions_abi.md#oak_functions_abi.md#read_request).
```

Comment:
Isn't it possible to link to the actual Rust symbol, instead of the generated page? Or does that not work because the import is not there?

Reply:
Not sure, but I'd try as is.